src/cascade/stats/estimation.py

```diff
@@ -47,3 +88,61 @@ def bounds_to_stdev(lower, upper):
         pd.Series: The standard deviation for that CI.
     """
     return (upper - lower) / (2 * 1.96)
+
+
```

Comment:
Your approach of computing all the options and masking them into the result is more clear than my masked calculations were. I like yours.